README.md

```diff
@@ -242,6 +242,14 @@ TraceWrapper.wrap_in_custom_span(config, "custom span") do |span|
 end
 ```
 
+The `trace_context:` keyword argument can be used to retrieve trace data:
+```ruby
+trace_context = { trace_id: '234555b04cf7e099', span_id: '234555b04cf7e099' }
+
+TraceWrapper.wrap_in_custom_span(config, "custom span", trace_context: trace_context) do |span|
```

Comment:
Is trace_contexst going to grow a lot? Seems little bit weird that we create a hash just for two keys?

Reply:
seems it should carry the sampling status also right?

Reply:
I guess my example here is not good.. 😅 

These are the minimum required keys and in reality, you are going to pass more keys like:
```ruby
trace_context = { 
  trace_id: '234555b04cf7e099', 
  span_id: '234555b04cf7e099', 
  sampled: 'true', 
  flags: '0'
}
```

I will update this example.

Reply:
ok in this case I would not use flags rather debug as that's more semantically compatible vs a relic of X-B3

Reply:
removed `flags` in 5f15a03